pkg/ipamplugin/ipam_plugin.go

```diff
@@ -109,6 +109,16 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
+	// We attach important attributes to the allocation.
+	attrs := map[string]string{
+		"node": nodename,
+	}
+	if epIDs.Pod != "" {
+		attrs["pod"] = epIDs.Pod
```

Comment:
Need to decide if we should just include the "wep" field and rebuild the pod / namespace from that, or if it is more useful to include the pod/namespace directly.